app/processing/statement/components/get-detailed-payments.js

```diff
@@ -0,0 +1,25 @@
+
+const moment = require('moment')
+const { convertToPounds } = require('../../../utility')
+
+const getDetailedPayments = async (calculation, paymentRequest, settlement) => {

```

Comment:
Also, weirdly I think for consistency this mapping should fall under a mapDetailedPayments function-file -- whatcha think?

Reply:
Again, every value needs to be string

Reply:
Mentioned in other comment but this function's params should be well defined

Reply:
As above